app/src/test/java/org/simple/clinic/summary/nextappointment/NextAppointmentUiRendererTest.kt

```diff
@@ -60,6 +60,7 @@ class NextAppointmentUiRendererTest {
     // then
     verify(ui).showAppointmentDate(LocalDate.parse("2018-01-01"))
     verify(ui).showChangeAppointmentButton()
+    verify(ui).hideAppointmentFacility()
```

Comment:
Now that we've updated this test to include this scenario, can we update the test name to reflect the same?
Like `when appointment is present and its same as assigned facility, then show appointment view`

Reply:
That is intentional. It's an implicit behaviour as far as we are concerned, but default the appointment facility is hidden. So we only explicitly have a test that verified its visibility.

Reply:
In later PR I will update the UI render logic to reflect that as well. Move the common hidden functions as implict.